### PR TITLE
feat/add arabic

### PR DESCRIPTION
**Pull request by ahmaxed**
this pr adds the logic to add rtl class to rtl languages and style the layout accrodingly.

### CODE REVIEW

**Comment by raisedadead** [comment]
Pro tip: start with a draft PR (not the same as marking draft after the fact) which helps with reducing notification noise.

**Comment by scissorsneedfoodtoo** [comment]
This is looking great. Once I change `LOCALE_FOR_UI` to `arabic`, all the views (landing, post/page, author, tag, search) shift to the RTL layout.

I noticed a couple of minor things while going through this, which I'll leave as review comments.

File: src/_includes/assets/css/screen.css [hunk 1]
```diff
@@ -2392,3 +2393,64 @@ a.banner:hover span {
     justify-content: space-between;
   }
 }
+
+/* 16. RTL layout
```
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]:
```suggestion
/* 17. RTL layout
```

File: src/_includes/assets/css/screen.css [hunk 2]
```diff
@@ -2392,3 +2393,64 @@ a.banner:hover span {
...
+  padding-left: 15px;
+}
+
+@media (min-width: 768px) {
+  .rtl-layout .post-card .post-card-content-link {
+    padding: 0 30px 0 0px;
+  }
+  .rtl-layout .post-card .post-card-meta {
+    padding: 0 34px 0 0px;
+  }
```
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]:
```suggestion
  .rtl-layout .post-card .post-card-content-link {
    padding: 0 30px 0 0px;
  }

  .rtl-layout .post-card .post-card-meta {
    padding: 0 34px 0 0px;
  }
```

File: src/_includes/assets/css/screen.css [hunk 3]
```diff
@@ -2392,3 +2393,64 @@ a.banner:hover span {
...
+}
+
+@media (max-width: 999px) {
+  .rtl-layout .site-nav-middle {
+    margin-left: auto;
+  }
+  .rtl-layout .site-nav-middle {
+    margin-right: 0;
+  }
```
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]:
```suggestion
  .rtl-layout .site-nav-middle {
    margin-left: auto;
  }

  .rtl-layout .site-nav-middle {
    margin-right: 0;
  }
```

File: src/_includes/layouts/default.njk [hunk 4]
```diff
@@ -113,8 +113,12 @@
...
         {{ codeinjection_head | safe if codeinjection_head }}
     </head>
-
-    <body class="home-template">
+    
+    {% if site.lang === 'ar' or site.lang === 'fa' or site.lang === 'ur' or site.lang ===  'he'%}
+        <body class="home-template rtl-layout">
+    {% else %}
+        <body class="home-template ">
+    {% endif %}
```
Comment (scissorsneedfoodtoo) [inline src/_includes/layouts/default.njk]: It's not explicitly stated in the Nunjucks docs, but we can simplify this a bit with the `in` statement, which works similarly to the JS `includes()` method in this context:

```suggestion
    {% if site.lang in ['ar', 'fa', 'ur', 'he'] %}
        <body class="home-template rtl-layout">
    {% else %}
        <body class="home-template">
    {% endif %}
```

**Review by scissorsneedfoodtoo — APPROVED** [review]
Did some quick testing with one of the articles on the Arabic Ghost instance and everything LGTM 👍 👍

@ahmadabdolsaheb, is this okay to merge as-is? Or will this PR also include the localization?

**Comment by ahmaxed** [comment]
@scissorsneedfoodtoo, once this is merged, I will point Islam to the localization files and ask him to submit a separate pr.

**Comment by scissorsneedfoodtoo** [comment]
Awesome! Sounds great. Will merge this now.

**Pull request merged**